internal/cmd/base/base.go

```diff
@@ -220,30 +226,39 @@ func (c *Command) Client(opt ...Option) (*api.Client, error) {
 
 	switch {
 	case c.FlagRecoveryConfig != "":
-		wrapper, err := wrapper.GetWrapperFromPath(c.FlagRecoveryConfig, "recovery")
+		wrapper, cleanupFunc, err := wrapper.GetWrapperFromPath(
```

Comment:
Reviewers: take a look at this file. This resolves a longstanding issue around finalization of wrappers instantiated as part of a command call (in order to handle any recovery flows in the API) where we weren't ever finalizing. If you look at the comment you can see that the change here was first theorized long ago (storing the function in the command) so this is just a matter of putting that idea in practice. The generated files then ensure to call this function if it exists.